vnext/Microsoft.ReactNative/Views/TextViewManager.cpp

```diff
@@ -140,7 +144,30 @@ class TextShadowNode final : public ShadowNodeBase {
     Super::RemoveChildAt(indexToRemove);
   }
 
+  int64_t GetReactTagAtPoint(const winrt::Point &point) {
+    if (pressableCount > 0) {
+      const auto hitTarget = VirtualTextShadowNode::HitTest(*this, point, /* hasPressableParent = */ false);
+      if (hitTarget != nullptr) {
+        auto inlineTag = GetTag(hitTarget);
+        if (inlineTag != -1) {
+          if (auto uiManager = GetNativeUIManager(GetViewManager()->GetReactContext()).lock()) {
+            const auto node = static_cast<ShadowNodeBase *>(uiManager->getHost()->FindShadowNodeForTag(inlineTag));
+            // React Native does not support events targeted to raw text nodes.
+            // Get the parent tag instead.
+            if (!std::wcscmp(node->GetViewManager()->GetName(), L"RCTRawText")) {
+              inlineTag = node->GetParent();
```

Comment:
nit, prefer == 0 to distinguish from comparison apis that return a bool (so that it doesn't read as "`!comparesAsEqual(...)`"